package.json

```diff
@@ -6,8 +6,10 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
+    "compile": "tsc --project tsconfig.prod.json",
     "setup": "gulp sass copy-files",
+    "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
```

Comment:
I think this should be inverted, to start and start-dev, so that the environments startup quicker as well

Reply:
I think you are right, I had the same thought. Then possibly just gulp would use `ts-dev`.